cypress/release.ts

```diff
@@ -1,3 +1,5 @@
+/* eslint cypress/no-unnecessary-waiting: 1 */
```

Comment:
Wish we actually had this as a rule, I'd rather have a warning (which acts as a todo itself, unless explicitly muted) or an ESLint rule that automatically surpasses any errors/warning on the line following TODO and makes it a warning. Otherwise, most TODOs get lost.